airflow-core/src/airflow/plugins_manager.py

```diff
@@ -406,21 +409,24 @@ def initialize_fastapi_plugins():
     """Collect extension points for the API."""
     global plugins
     global fastapi_apps
+    global fastapi_root_middlewares
 
-    if fastapi_apps:
+    if fastapi_apps and fastapi_root_middlewares:
```

Comment:
```suggestion
    if fastapi_apps or fastapi_root_middlewares:
```

I think?

Reply:
And shouldn't this also be 
```suggestion
    if fastapi_apps is not None and fastapi_root_middlewares is not None:
```

in case it's loaded but the lists are empty?

Reply:
Nice catch, indeed, thanks. 

Updated.